aat/src/resources/features/F-023/S-009.td.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Must return a negative response on non existent jurisdiction ID",
+  "_guid_": "S-009",
+  "_extends_": "F-023_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD - wants to see a list of valid documents they can print",
+    "contains a non existent jurisdictionID"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "jid": "ABCDEFGH",
+      "ctid": "FT_MasterCaseType",
+      "cid": "1111222233334444"
+     }
+    },
+
+  "expectedResponse": {
+    "responseCode": 403,
+    "responseMessage": "Forbidden",
```

Comment:
Please extent the common 403 response object. That will bring the 2 lines here and the headers below.

Reply:
no need for the headers section then.

Reply:
done